elevator_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

floor_position_controller:
  type: effort_controllers/JointPositionController
  joint: floor
  pid: {p: 100000.0, i: 5.0, d: 1000.0}

car_door_left_controller:
  type: effort_controllers/JointPositionController
  joint: car_door_left_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}

car_door_right_controller:
  type: effort_controllers/JointPositionController
  joint: car_door_right_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}


lobby_door_left_controller:
  type: effort_controllers/JointPositionController
  joint: lobby_door_left_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}

lobby_door_right_controller:
  type: effort_controllers/JointPositionController
  joint: lobby_door_right_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}


first_door_left_controller:
  type: effort_controllers/JointPositionController
  joint: first_door_left_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}

first_door_right_controller:
  type: effort_controllers/JointPositionController
  joint: first_door_right_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}


second_door_left_controller:
  type: effort_controllers/JointPositionController
  joint: second_door_left_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}

second_door_right_controller:
  type: effort_controllers/JointPositionController
  joint: second_door_right_joint
  pid: {p: 100.0, i: 0.01, d: 10.0}
